src/formatDate.js

```diff
@@ -8,7 +8,70 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  if (date !== undefined) {
```

Comment:
The check for `date !== undefined` is unnecessary because the task assumes that the `formatDate` function will always be called with correctly formatted arguments. You can safely remove this check.